src/Microsoft.Diagnostics.Monitoring.Options/EgressCategoryOptions.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Monitoring.WebApi;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.Egress
+{
+    /// <summary>
+    /// Egress provider options for external egress providers.
+    /// </summary>
+    internal sealed partial class EgressCategoryOptions
+    {
+        public object ConfigurationSection { get; set; }
```

Comment:
From what I can tell, this property isn't used (yet?). I would also ask you to not just store the configuration in the options type since that is leaking configuration through options (separation of responsibilities). Just pull out what is needed from configuration and store it on the options instance.